build.sh

```diff
@@ -440,14 +440,25 @@ __CleanBuild=0
 __VerboseBuild=0
 __SignTypeReal=""
 __CrossBuild=0
-__ClangMajorVersion=3
-__ClangMinorVersion=5
 __MSBuildPath=$__ProjectRoot/Tools/MSBuild.exe
 __NuGetPath="$__PackagesDir/NuGet.exe"
 __DistroName=""
 __cmakeargs=""
 __OfficialBuildIdArg=
 __SkipGenerateVersion=0
+__ClangMajorVersion=3
+__ClangMinorVersion=5
+while :; do
+    if [ $# -le 0 ]; then
+        break
+    fi
+    lowerI="$(echo $1 | awk '{print tolower($0)}')"
+    if [ $lowerI  == cross ]; then
```

Comment:
> As for your second scenario, I now think that instead of what I've suggested, we should set the clang version after the existing loop parsing the command line if the __CrossBuild value is 1

@janvorli  ,  It's better.  I will  update the PR with $__CrossBuild variable.